src/automation/tasks/guidellm.py

```diff
@@ -1,15 +1,17 @@
 from automation.tasks import BaseTask
-from automation.configs import DEFAULT_DOCKER_IMAGE
+from automation.configs import DEFAULT_DOCKER_IMAGE, DEFAULT_RESEARCH_BRANCH
 from typing import Optional, Sequence
 import os
 
 DEFAULT_SERVER_WAIT_TIME = 600 # 600 seconds = 10 minutes
-GUIDELLM_PACKAGE = "git+https://github.com/neuralmagic/guidellm.git@http_backend"
+GUIDELLM_PACKAGE = "git+https://github.com/neuralmagic/guidellm.git"
 
 class GuideLLMTask(BaseTask):
 
     guidellm_packages = [
-        "vllm",
+        "huggingface-hub==0.34.3",
+        "triton==3.3.1",
+        "vllm==0.10.0",
```

Comment:
Let us not pin these versions in the repo. This is likely a transitory state and should not land on main